src/Jackett.Common/Definitions/letseed.yml

```
@@ -0,0 +1,215 @@
+    - {id: 13, cat: Movies/Other, desc: "Yerli Yapımlar"}
+    - {id: 32, cat: Audio, desc: "Müzik"}
+    - {id: 35, cat: Audio/Other, desc: "Diskografi"}
+    - {id: 34, cat: Audio/Foreign, desc: "Yabancı"}
+    - {id: 33, cat: Audio/Other, desc: "Yerli"}
+    - {id: 28, cat: PC/Games, desc: "Oyun"}
+    - {id: 29, cat: PC/Games, desc: "PC"}
+    - {id: 31, cat: Console/PS4, desc: "Playstation"}
+    - {id: 30, cat: Console/XBox, desc: "XBOX"}
+    - {id: 19, cat: Other, desc: "Platform Sunumları"}
+    - {id: 21, cat: Other, desc: "AMZN"}
+    - {id: 23, cat: Other, desc: "BluTv"}
+    - {id: 22, cat: Other, desc: "DSNP"}
+    - {id: 24, cat: Other, desc: "Exxen"}
+    - {id: 26, cat: Other, desc: "GAIN"}
+    - {id: 20, cat: Other, desc: "Netflix"}
+    - {id: 42, cat: Other, desc: "Tabii"}
+    - {id: 25, cat: Other, desc: "TOD"}
+    - {id: 27, cat: Other, desc: "TV+"}
+    - {id: 36, cat: PC, desc: "Program"}
+    - {id: 37, cat: PC, desc: "İşletim Sistemi"}
+    - {id: 38, cat: PC/Mac, desc: "Mac"}
+    - {id: 39, cat: PC, desc: "Windows"}
+    - {id: 40, cat: TV/Sport, desc: "Spor"}
+    - {id: 41, cat: TV/Sport, desc: "Spor Sunumları"}
+    - {id: 17, cat: Other, desc: "VIP Sunumlar"}
+    - {id: 18, cat: Other, desc: "VIP Sunumlar"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: freeleech
+    type: checkbox
+    label: Filter freeleech only
+    default: false
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: added
+    options:
+      added: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+  - name: info_tpp
+    type: info
+    label: Results Per Page
+    default: For best results, change the <b>Torrents per page:</b> setting to <b>40</b> on your account profile. The Default is <i>30</i>.
+  - name: info_activity
+    type: info
+    label: Account Inactivity
+    default: "To keep records updated regularly, all inactive accounts will be deleted after 60 days of inactivity."
+
+login:
+  path: login.php
+  method: form
+  form: form[action="takelogin.php"]
+  captcha:
+    type: image
+    selector: img#regimage
+    input: imagestring
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+    logout: ""
+  error:
+    - selector: '[http-equiv="refresh"][content*="/login.php"]'
+      # TODO: This tracker uses `http-equiv="refresh"` for all login form submissions,
+      # valid or invalid, and uses the `error=#` query parameter to tell the subsequent
+      # login form what error message to display. How should this definition yield a
+      # useful error message?
```

```suggestion
  error:
    - selector: table:has(a[href*="login.php?error=1"])
      message:
        text: "ERROR: Incorrect username!"
    - selector: table:has(a[href*="login.php?error=4"])
      message:
        text: "ERROR: Incorrect password!"
    - selector: table:has(a[href*="login.php?error="])
      message:
        text: "ERROR: Something went wrong during login!"
```
the takelogin.php page has an error code we can use, similar to futuretorrent.yml

---

I'll integrate this change when I get a moment to fire up the VPN and run though all the error conditions.

---

I think I misunderstood when this indexer's web UI uses proper 3## redirects and when it's using `http-equiv="refresh"`. In the case of POSTing the login form when there's an error, it *does* do a proper `302.Redirect` so we *should* be able to use this selector. My test bed has been banned again despite rotating my VPN connection, however, so I can't confirm this right now. Someone else can test it or I'll try again soon another day

---

> I think I misunderstood when this indexer's web UI uses proper 3## redirects and when it's using `http-equiv="refresh"`.

Nope, it's *both*! I just did a test in the Prowlarr edit form where I entered the wrong captcha and I got a proper `302.Redirect` response. Then I did a test where I entered the right captcha and the wrong password and I got a `http-equiv="refresh"` response. I don't know if it's stable depending on the error, but I'm not going to try and confirm that given how easy it is to hit a ban. Trying to work it out somehow.

---

Thinking about this further, it would be good for a web scraping framework to treat a `http-equiv="refresh"` "redirect" as equivalent to a proper `302.Redirect` by default, probably with an option to disable. Maybe worth a feature request?

---

Ok, I used both the selector for proper `302.Redirect` redirects *and* the 4 selectors for `http-equiv="refresh"` "redirects" and it worked for me when I tested:

- wrong captcha
- wrong passphrase
- wrong usename
- correct credentials and captcha

In this round of testing, *none* of the login form `POST`s got a proper `302.Redirect` response, so my best guess is that maybe the indexer's web UI switches to `http-equiv="refresh"` "redirects" after a failure. Regardless, it's definitely not stable for the error type. IOW, the same error can result in either response type.

At any rate, according to my testing this is now resolved, but it feels so fragile I'm leaving this conversation unresolved until someone else can test.